client/my-sites/plans/p2-plans-main.jsx

```diff
@@ -1,30 +1,27 @@
-import { PLAN_P2_FREE, PLAN_P2_PLUS } from '@automattic/calypso-products';
 import { localize } from 'i18n-calypso';
 import { Component } from 'react';
 import { connect } from 'react-redux';
 import QueryPlans from 'calypso/components/data/query-plans';
 import QuerySitePlans from 'calypso/components/data/query-site-plans';
 import QuerySites from 'calypso/components/data/query-sites';
-import PlanFeatures from 'calypso/my-sites/plan-features';
+import PlansFeaturesMain from '../plans-features-main';
 
 export class P2PlansMain extends Component {
 	render() {
-		const { selectedFeature, selectedPlan, redirectTo, siteId } = this.props;
+		const { siteId } = this.props;
 
 		return (
 			<>
 				<QueryPlans />
```

Comment:
Unsure if these queries are necessary 🤔 If anyone happens to know, please feel free to chime in

Reply:
They are probably not needed as the `plans-features-main` queries the same there. I think I'd keep them just to be safe. `plans-features-main` will be going through changes and might affect which queries are called?

Reply:
Makes sense 🙂

In that case I'll leave them for now